src/pages/login.js

```diff
@@ -1,5 +1,81 @@
+import Background from "../assets/universe_background.jpeg";
+import Logo from "../assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+import { post } from "../Utils";
+
 function Login() {
-  return <h1>Login</h1>;
+  const navigate = useNavigate();
+
+  const handleSubmit = async (event) => {
+    const data = new FormData(event.target);
+    const mail = data.getElementById("email").value;
```

Comment:
If using FormData, and if the inputs have `name` attributes, you can just use the `data.get("email")` method

Reply:
Otherwise, looks good!